contracts/src/v0.8/functions/dev/v1_X/accessControl/TermsOfServiceAllowList.sol

```diff
@@ -17,6 +17,7 @@ contract TermsOfServiceAllowList is ITermsOfServiceAllowList, IAccessController,
 
   /// @inheritdoc ITypeAndVersion
   string public constant override typeAndVersion = "Functions Terms of Service Allow List v1.1.0";
+  address public s_previousToSContract;
```

Comment:
Style guide dictates that this should be internal/private. If a getter is needed it should be explicitly added.